modules/scripts/startup-script/README.md

```diff
@@ -162,6 +162,24 @@ sudo journalctl -u google-startup-scripts.service
   use: [homefs, startup]
 ```
 
+```yaml
```

Comment:
We should add a line of context as to what this example is showing.

Reply:
done.